libs/proxy/tokio-postgres2/src/error/mod.rs

```diff
@@ -86,6 +86,27 @@ pub struct DbError {
 }
 
 impl DbError {
+    pub fn new(code: SqlState, message: String) -> Self {
```

Comment:
nit: since this is intended only for testing and probably shouldn't be used in other cases, let's rename this to `new_test_error`.